### PR TITLE
Add new message types for image storage usage event

### PR DESCRIPTION
**Pull request by simao**
add scalatest dependency to common-test

This ticket needs changes to 3 projects:

https://github.com/advancedtelematic/treehub/pull/28
https://github.com/advancedtelematic/auditor/pull/5
https://github.com/advancedtelematic/rvi_sota_server/pull/708

### CODE REVIEW

File: core/src/test/scala/org/genivi/sota/core/PackageResourceWordSpec.scala [hunk 1]
```diff
@@ -11,8 +11,9 @@ import akka.http.scaladsl.model.Uri
 import akka.http.scaladsl.server.MalformedQueryParamRejection
 import akka.http.scaladsl.testkit.ScalatestRouteTest
 import eu.timepit.refined.api.Refined
+import org.genivi.sota.DefaultPatience
 import org.genivi.sota.marshalling.CirceMarshallingSupport
-import org.genivi.sota.core.data.{Package => DataPackage}
+import org.genivi.sota.core.data.{Package ⇒ DataPackage}
```
Comment (Danten) [inline core/src/test/scala/org/genivi/sota/core/PackageResourceWordSpec.scala]: Unicode again

File: core/src/test/scala/org/genivi/sota/core/PackagesResourceSpec.scala [hunk 2]
```diff
@@ -11,7 +11,7 @@ import java.net.URI
 import org.genivi.sota.data.PackageId._
 import org.genivi.sota.marshalling.CirceMarshallingSupport._
 import io.circe.generic.auto._
-import org.genivi.sota.core.data.{Package => DataPackage}
+import org.genivi.sota.core.data.{Package ⇒ DataPackage}
```
Comment (Danten) [inline core/src/test/scala/org/genivi/sota/core/PackagesResourceSpec.scala]: Unicode

File: common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala [hunk 3]
```diff
@@ -53,6 +53,8 @@ object Messages {
     namespace: Namespace,
     packageId: PackageId) extends BusMessage
 
+  final case class ImageStorageUsage(namespace: Namespace, timestamp: Instant, byteCount: Long) extends BusMessage
```
Comment (Danten) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala]: This is formatted differently from the other messages.

File: common-test/src/main/scala/org/genivi/sota/DefaultPatience.scala [hunk 4]
```diff
@@ -1,8 +1,9 @@
-/**
+/*
  * Copyright: Copyright (C) 2016, ATS Advanced Telematic Systems GmbH
- * License: MPL-2.0
+ *  License: MPL-2.0
```
Comment (Danten) [inline common-test/src/main/scala/org/genivi/sota/DefaultPatience.scala]: Was the change to the header intentional?

File: common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala (hunk 3 above)
Comment (simao) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala]: Yes, but the others are wrong ;)

But seriously, the other message formatting was all changed in a recent commit for some reason and changed the whole formatting to something I would not consider better.

Comment (Danten) [inline common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala]: Yeah they got changed by @rdanitz  when he added the `DeviceActivated` event. Anyway you clearly have stronger opinions  than me which one is correct so I let you decide, I just want to reach some form of consistency. Hopefully this is not something that will be changed for every commit... ;P

**Pull request merged**